Framework/DataObjects/inc/MantidDataObjects/SpecialWorkspace2D.h

```diff
@@ -47,6 +47,8 @@ class MANTID_DATAOBJECTS_DLL SpecialWorkspace2D : public Workspace2D {
   @return Standard string name  */
   const std::string id() const override { return "SpecialWorkspace2D"; }
 
+  bool IsDetectorIDMappingEmpty() const { return detID_to_WI.empty(); }
```

Comment:
This one should change to pascalCase too

Reply:
Done.